lib/browser.ts

```diff
@@ -668,6 +693,12 @@ export class ProtractorBrowser extends Webdriver {
       return 'Protractor.get(' + destination + ') - ' + str;
     };
 
+    if (this.bpClient) {
```

Comment:
Hm, why is this necessary? Because we do executeScript calls, and BpClient isn't smart enough  to know that it doesn't need to wait for those?